packages/unless/src/index.ts

```diff
@@ -0,0 +1,89 @@
+import type { Request, Response, NextFunction } from '@tinyhttp/app'
+
+//Options for the middleware
+export interface UnlessMiddlewareOptions {
+  method?: string | string[] //methods to compare
+  path?: string | RegExp | (string | RegExp | PathObject)[] //paths to compare
+  ext?: string | string[] //last part of endpoint to compare
+}
+
+//Optional path object
+interface PathObject {
+  url: string
+  methods: string[]
+}
+
+//Middleware interface
+type Middleware = (req: Request, res: Response, next?: NextFunction) => any
```

Comment:
I forgot to re-export `Middleware` type in `app` package my bad

now it's re-exported so u just need to do `import type { Middleware } from '@tinyhttp/app'`